error_113_test.go

```diff
@@ -0,0 +1,104 @@
+// +build go1.13
+
+package errorx
+
+import (
+	"errors"
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestErrorUnwrap(t *testing.T) {
+	t.Run("Trivial", func(t *testing.T) {
+		err := testType.NewWithNoMessage()
+		unwrapped := errors.Unwrap(err)
+		require.Nil(t, unwrapped)
+	})
+
+	t.Run("Wrap", func(t *testing.T) {
+		err := testTypeBar1.Wrap(testType.NewWithNoMessage(), "")
+		unwrapped := errors.Unwrap(err)
```

Comment:
This is where unrelatedness between `Wrap` and `Unwrap` hits particularly hard :(